docs/onboarding_to_cluster.md

```diff
@@ -0,0 +1,151 @@
+# Onboarding to a cluster
+
+Currently we support 2 ways to request/perform onboarding to any supported clusters:
+
+1. Onboarding request - please use the **Onboarding to Cluster** issue template in [operate-first/support](https://github.com/operate-first/support) repository.
+2. Opening a PR with the desired changes. The rest of this doc focuses on this second option. We'd gladly welcome your contributions.
+
+## Prerequisites
+
+1. Please install our [toolbox](https://github.com/operate-first/toolbox) to have the developer setup ready automatically for you.
+2. Or you can install the tools manually. You'll need [kustomize](https://kustomize.io/), [sops](https://github.com/mozilla/sops) and [ksops](https://github.com/viaduct-ai/kustomize-sops).
```

Comment:
The numbering here seems to imply that they should do 1, then 2, but it's actually 1 or 2, to give this impression I would format it like so: 

> You will need pre-requisite tools to follow along with this doc, please do either of the following: 
> - Install our toolbox to have the developer setup ready automatically for you.
> - Install the tools manually. You'll need kustomize, sops and ksops.